examples/preprocessing/plot_csd.py

```diff
@@ -0,0 +1,94 @@
+"""
+===============================================
+Analyze Data Using Current Source Density (CSD)
+===============================================
+
+This script shows an example of how to use CSD. CSD
+takes the spatial Laplacian of the sensor signal
+(derivative in both x and y). The spatial derivative
+reduces point spread. CSD transformed data have a
+sharper or more distinct topography, reducing the
+negative impact of volume conduction.
```

Comment:
why 2 examples ?

Reply:
I thought the parameter search example might be helpful to users but we can delete it.